src/compiler/scala/tools/nsc/typechecker/Contexts.scala

```diff
@@ -947,10 +947,10 @@ trait Contexts { self: Analyzer =>
      *  if any such symbol is accessible from this context.
      */
     def importedAccessibleSymbol(imp: ImportInfo, name: Name): Symbol =
```

Comment:
so this method is only called from `collectImplicitImports`, in which case you want to have `record = false`. to me it seems a little bit more clear to push `record = false` out to the callsite within `collectImplicitImports`. the method name here (`importedAccessibleSymbol`) doesn't tell me why it should or should not record the import.